docs/sources/panels/transformations/_index.md

```diff
@@ -29,11 +29,11 @@ Before you can configure and apply transformations:
 
 - You must have entered a query and returned data from a data source. For more information on queries, refer to [Queries]({{< relref "../queries.md" >}}).
   
-- You must have applied a visualization that supports queries. Examples are:
+- You must have applied a visualization that supports your queries. Different transformations can result in a data structure that only some visualizations can render. For example, the visualization cannot show reduced data (that results from the Reduce transform). In case  your current visualization does not show you what you expect after applying a transform, switch to the Table visualization to check the result. For more information on the different types of visualizations, refer to:
```

Comment:
```suggestion
- You must have applied a visualization that supports your query result and chosen transforms. Different transformations can result in a data structure that only some visualizations can render. For example, the visualization cannot show reduced data (that results from the Reduce transform). In case  your current visualization does not show you what you expect after applying a transform, switch to the Table visualization to check the result. For more information on the different types of visualizations, refer to:
```

Reply:
I suggest breaking up this paragraph. General rule is three sentences or fewer per paragraph, 25 words or less per sentence.

Reply:
Also, you can probably replace "in case" with "if."

Reply:
Keeping the content into one paragraph, as it is a bullet list.